### PR TITLE
AC Test Fix - LRAC-10836 | master

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-10836
This has been failing on CI more than it passes. Main reason being the CI runs near the end of day where the day the CI grabs the date is from yesterday. The times it passes possibly from the CI run finishes before the day ends

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase [hunk 1]
```diff
@@ -108,7 +108,7 @@ definition {
 		}
 
 		task ("Get current date") {
-			var dayNum = DateUtil.getCurrentDate();
+			var dayNum = DateUtil.getDate("1");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:111]: Hey @joshitagaki, My question is just curiosity. Passing the value to the macro with "1" makes it return what? Could you explain how this works?

Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:111]: Sure! The getDate() with no parameter is automatically "0" which is the current date. Changing to "1" will return tomorrows date. You can see the testray run here: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1658176813&testrayRunId=1658177352. It fails but it grabs tomorrows date

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:111]: Ohhh ok, very good Josh. Thank you for the explanation. Yes, this way we will always have the date of tomorrow which is when the build is executed. Congratulations! 😄

**Review by ViniLopes20 — APPROVED** [review]
LGTM, Congratulations! 😄

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase (hunk 1 above)
Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:111]: wooow, I didn't that! thank you josh!

**Review by manuferreira — APPROVED** [review]
LGTM! Very nice Josh. :nerd_face:

**Comment by christianstokes** [comment]
Hey @joshitagaki with this change, is it only going to work if the test runs at a specific time?

Would the test fail during other times? Like if I run it locally during the day or to test a PR?

**Comment by joshitagaki** [comment]
> Hey @joshitagaki with this change, is it only going to work if the test runs at a specific time?
> 
> Would the test fail during other times? Like if I run it locally during the day or to test a PR?

Yes it would fail if you were to try and run it other times. This would fix the runs on CI since those mainly run at the end of the day

**Comment by joshitagaki** [comment]
Testing fix. Checks if current date fails will check tomorrows date.

**Comment by joshitagaki** [comment]
Tetray run with changes: https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1667388654&testrayRunId=1667394662

**Comment by joshitagaki** [comment]
ci:test:sf

**Comment by joshitagaki** [comment]
New PR: https://github.com/liferay-qa/liferay-portal/pull/1768

**Pull request closed without merging**